crates/ironrdp-rdpdr/src/lib.rs

```diff
@@ -30,42 +27,49 @@ use crate::pdu::efs::{
 pub struct Rdpdr {
     /// TODO: explain what this is
     computer_name: String,
+    /// See [`CapabilityMessage`]
```

Comment:
Nit: not a big deal, but it seems to me that this doc string is of little use (not an explanation and we can already jump to the definition of CapabilityMessage without the link it adds)

Reply:
001d937d366b978775b62aa9455105ebd60bfc6c